### PR TITLE
Change ingress path for flower to fix livenessProbe

### LINKED ISSUE #17: Flower Pod goes into CrashLoopBackoff
kubectl get events:

`kubectl events says: spec.containers{airflow-flower}   Warning   Unhealthy   kubelet, k8s-development-41339263-2   Liveness probe failed: HTTP probe failed with statuscode: 404
`

The probe is defined as path: “{{ .Values.ingress.flower.path }}/” and the value in values.yaml is /flower

So the probe checks `localhost:5555/flower`, which returns 404. Changing the value in the values.yaml to empty string (“”) fixes the probe.

### PR DESCRIPTION
**Pull request by rolanddb**
Fixes #17

### CODE REVIEW

File: airflow/templates/deployments-flower.yaml [hunk 1]
```diff
@@ -36,7 +36,9 @@ spec:
               containerPort: 5555
               protocol: TCP
           args: ["flower"]
-          livenessprobe:
+          livenessProbe:
             httpGet:
               path: "{{ .Values.ingress.flower.path }}/"
```
Comment (gsemet) [inline airflow/templates/deployments-flower.yaml]: no, here you need to remove this "{{ .Values.ingress.flower.path }}"

File: airflow/values.yaml [hunk 2]
```diff
@@ -115,7 +115,8 @@ ingress:
     ## request to have been be at the root.
     ## That's why we need a reverse proxy/load balancer that is able to strip the path
     ## NOTE: do NOT keep trailing slash. For root configuration, set and empty string
-    path: /flower
+    ## NOTE: this is also used by livenessProbe
+    path: /
```
Comment (gsemet) [inline airflow/values.yaml]: that's the problem. Livenessprobe should not use this path.

You can set the default to "/", ok, but do not add "NOTE: this is also used by livenessProbe"

**Comment by rolanddb** [comment]
@gsemet Thanks for the review, I made the changes and tested on our k8s cluster.

P.s. slightly related: The deployment for web also points to Values.ingress.web.path, should that be changed as well?

**Comment by gsemet** [comment]
@rolanddb no, because contrary to Flower that requires URL rewrite to work, Airflow is able to handle properly incoming URLs under the path.

I know this is weird, and I tried to explain it in the value.yaml, but its tricky to explain

**Pull request merged**